### DISCUSSION TITLE
Is GitHub Insider only for enterprises?

### DISCUSSION THREAD

**Discussion by rfalanga** [community/community#134365] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

At work I get the GitHub Insider Newsletter. I'd like to subscribe to that at home, but when I tried searching for it the results suggested it was only for enterprises/corporations.

Is that true?

**Comment by Shreyas719** [comment]
No

**Comment by queenofcorgis** [comment]
The GitHub Insider Newsletter is for all developers, engineers, and tech professionals. Could you share more about where you received the messaging that it was only for enterprises/corporations?

**Comment by rfalanga** [comment]
When I go to the page to sign up for the [Insider newsletter](https://resources.github.com/newsletter/) it says specifically, **Work Email**. That tells me it is not for the unwashed masses.

**Reply by queenofcorgis** [reply, marked as answer]
@rfalanga there are a number of factors that go into the wording of that, but the content is very much for everyone using GitHub so I encourage you to subscribe!

**Reply by rfalanga** [reply]
@queenofcorgis OK, then I'll ignore the wording telling me that GitHub Insider is only for Enterprise developers, and I'll subscribe. Thank you.

**Reply by queenofcorgis** [reply]
@rfalanga I hope you enjoy the content!
